src/data/symbol/symbol.tsv

```diff
@@ -1957,85 +2056,85 @@ POS	CHAR	Reading (space separated)	description	additional description	category	m
 記号	𛄡	へんたいがな いぇ	変体仮名
 記号	𛄡	へんたいがな え	変体仮名
 記号	𛄢	へんたいがな う	汙の変体仮名
-記号	あ゙	あ だくてん	濁点付き仮名
-記号	ぁ゙	あ ぁ だくてん	濁点付き仮名
-記号	い゙	い だくてん	濁点付き仮名
-記号	ぃ゙	い ぃ だくてん	濁点付き仮名
-記号	ぅ゙	う ぅ だくてん	濁点付き仮名
-記号	え゙	え だくてん	濁点付き仮名
-記号	ぇ゙	え ぇ だくてん	濁点付き仮名
-記号	お゙	お だくてん	濁点付き仮名
-記号	ぉ゙	お ぉ だくてん	濁点付き仮名
-記号	っ゙	づ っ だくてん	濁点付き仮名
-記号	な゙	な だくてん	濁点付き仮名
-記号	に゙	に だくてん	濁点付き仮名
-記号	ぬ゙	ぬ だくてん	濁点付き仮名
-記号	ね゙	ね だくてん	濁点付き仮名
-記号	の゙	の だくてん	濁点付き仮名
-記号	ま゙	ま だくてん	濁点付き仮名
-記号	み゙	み だくてん	濁点付き仮名
-記号	む゙	む だくてん	濁点付き仮名
-記号	め゙	め だくてん	濁点付き仮名
-記号	も゙	も だくてん	濁点付き仮名
-記号	や゙	や だくてん	濁点付き仮名
-記号	ゃ゙	や ゃ だくてん	濁点付き仮名
-記号	ゆ゙	ゆ だくてん	濁点付き仮名
-記号	ゅ゙	ゆ ゅ だくてん	濁点付き仮名
-記号	よ゙	よ だくてん	濁点付き仮名
-記号	ょ゙	よ ょ だくてん	濁点付き仮名
-記号	ら゙	ら だくてん	濁点付き仮名
-記号	り゙	り だくてん	濁点付き仮名
-記号	る゙	る だくてん	濁点付き仮名
-記号	れ゙	れ だくてん	濁点付き仮名
-記号	ろ゙	ろ だくてん	濁点付き仮名
-記号	わ゙	わ だくてん	濁点付き仮名
-記号	ゐ゙	ゐ だくてん	濁点付き仮名
-記号	ゑ゙	ゑ だくてん	濁点付き仮名
-記号	を゙	を だくてん	濁点付き仮名
-記号	ん゙	ん だくてん	濁点付き仮名
-記号	ア゙	あ だくてん	濁点付き仮名
-記号	ァ゙	あ ぁ だくてん	濁点付き仮名
-記号	イ゙	い だくてん	濁点付き仮名
-記号	ィ゙	い ぃ だくてん	濁点付き仮名
-記号	ゥ゙	う ぅ だくてん	濁点付き仮名
-記号	エ゙	え だくてん	濁点付き仮名
-記号	ェ゙	え ぇ だくてん	濁点付き仮名
-記号	オ゙	お だくてん	濁点付き仮名
-記号	ォ゙	お ぉ だくてん	濁点付き仮名
-記号	ッ゙	づ っ だくてん	濁点付き仮名
-記号	ナ゙	な だくてん	濁点付き仮名
-記号	ニ゙	に だくてん	濁点付き仮名
-記号	ヌ゙	ぬ だくてん	濁点付き仮名
-記号	ネ゙	ね だくてん	濁点付き仮名
-記号	ノ゙	の だくてん	濁点付き仮名
-記号	マ゙	ま だくてん	濁点付き仮名
-記号	ミ゙	み だくてん	濁点付き仮名
-記号	ム゙	む だくてん	濁点付き仮名
-記号	メ゙	め だくてん	濁点付き仮名
-記号	モ゙	も だくてん	濁点付き仮名
-記号	ヤ゙	や だくてん	濁点付き仮名
-記号	ャ゙	や ゃ だくてん	濁点付き仮名
-記号	ユ゙	ゆ だくてん	濁点付き仮名
-記号	ュ゙	ゆ ゅ だくてん	濁点付き仮名
-記号	ヨ゙	よ だくてん	濁点付き仮名
-記号	ョ゙	よ ょ だくてん	濁点付き仮名
-記号	ラ゙	ら だくてん	濁点付き仮名
-記号	リ゙	り だくてん	濁点付き仮名
-記号	ル゙	る だくてん	濁点付き仮名
-記号	レ゙	れ だくてん	濁点付き仮名
-記号	ロ゙	ろ だくてん	濁点付き仮名
-記号	ヷ	わ だくてん	濁点付き仮名
-記号	ヸ	ゐ だくてん	濁点付き仮名
-記号	ヹ	ゑ だくてん	濁点付き仮名
-記号	ヺ	を だくてん	濁点付き仮名
-記号	ン゙	ん だくてん	濁点付き仮名
-記号	か゚	びだくおん か が	鼻濁音
-記号	き゚	びだくおん き ぎ	鼻濁音
-記号	く゚	びだくおん く ぐ	鼻濁音
-記号	け゚	びだくおん け げ	鼻濁音
-記号	こ゚	びだくおん こ ご	鼻濁音
-記号	カ゚	びだくおん か が	鼻濁音
-記号	キ゚	びだくおん き ぎ	鼻濁音
-記号	ク゚	びだくおん く ぐ	鼻濁音
-記号	ケ゚	びだくおん け げ	鼻濁音
-記号	コ゚	びだくおん こ ご	鼻濁音
+記号	あ゙	あ あ" あ゛ だくてん	濁点付き仮名
```

Comment:
> If it's for Kana input,

Bingo.

> it might be preferred to modify other rule files.

Do you mean the <kbd>゛</kbd> key after a hiragana should generate U+3099 instead of U+309B?